_posts/patterns/2016-03-01-disclosure.md

```diff
@@ -0,0 +1,30 @@
+---
+layout:         pattern
+title:          Disclosure Component
+date:           2016-03-01 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- disclosure
+
+slug:           disclosure
+
+description:    Examples of disclosure component.
+
+info:           TBD
+---
+
+<h3 class="hd-6 example-set-hd">Disclosure Component</h3>
+<div class="example-set">
+    <div class='disclosure' data-expanded-text="Hide Description" data-collapsed-text="Show Description">
+        <button type="button" class="btn-brand disclosure-toggle">
```

Comment:
@dsjen we need to add some aria-* here.  First, we need to programmatically indicated the state of the disclosure with ```aria-expanded=true``` when the target is visible and ```false``` when its not.  Second we need to programmatically indicate what is being expanded or collapsed with ```aria-controls``` which takes an IDref.  For that, you will need to add an ID attribute to the sibling ```div``` (in this case).

I noticed that the rendered control already has ```aria-expanded``` but it is lacking the ```aria-controls```.  The markup example in the PL lacks both.